### DISCUSSION TITLE
Wants to contribute.

### DISCUSSION THREAD

**Discussion by BlossomBreeze25** [uaanchorage/GSoC#4] [General]
Respected sir / ma'am,
I am Ragini Devi Thakkar , 4th Semester BCA(AI) student, want to join your organization. i have knowledge about AI and languages like python , java . html , css . i am interested in this project : epilepsy seizures detection and it's affect rate on your brain and human body. As i am suffering from epilepsy and it is affected my learning abilities and thinking skills and some other health issue i faced from past several years. Your organization helps in this problems and I would love to contribute to your organization but could you please tell me how to get started?

Hoping to hear from you soon.

Regards,

Ragini Devi Thakkar.

**Comment by harshal-rembhotkar** [comment]
Respected sir/ma'am,
 I am Harshal Rembhotkar, a 4th-semester Engineering (Automobile) student. I am excited to join your organization. My tech stacks are Java, HTML, CSS, version control systems, and technologies such as Docker and Kubernetes. I am particularly interested in contributing to the Beehive project: "Integrated Community Health Metrics Framework for Behavioral Health to Supplement Healthcare Practice in Alaska." I am completely enthusiastic about taking part in GSOC 2024 in Alaska and giving my best in it! 
 Thank you..

Linkedin: https://www.linkedin.com/in/harshal-rembhotkar/
Github: https://github.com/Harshal-Rembhotkar

**Comment by onlyvanshk** [comment]
Respected Sir/Madam,

I hope this email finds you well. My name is Vansh Kumar, and I am a second-year computer science undergrad at GGSIPU.

While I am new to open-source contributions, I am proficient in HTML, CSS, JavaScript, React.js, Python, and SQL. I am eager to contribute to your organization and learn through meaningful collaboration. Could you kindly guide me on how to get started with GSoC 2025?

Looking forward to your response.

Warm regards,
Vansh Kumar
